website/docs/Pipelines/Exome_Germline_Single_Sample_Pipeline/README.md

```diff
@@ -63,10 +63,55 @@ Check out the workflow [Methods](./exome.methods.md) to get started!
 ## Workflow Outputs
 
 - CRAM, CRAM index, and CRAM MD5
-- GVCF and its GVCF index
+- [Reblocked](https://gatk.broadinstitute.org/hc/en-us/articles/360037593171) GVCF and its GVCF index
 - BQSR report
 - Summary metrics; to read more about any particular metric, you can search the metric using the [GATK documentation search](https://gatk.broadinstitute.org/hc/en-us/categories/360002302312)
 
+### Reblocking
+Reblocking is a process that compresses a HaplotypeCaller GVCF by merging homRef blocks according to new genotype quality (GQ) bands. 
+
+As of November 2021, reblocking is a default task in the Exome pipeline. To skip reblocking, add the following to the workflow's input configuration file (JSON):
+
+```WDL
+"WholeGenomeGermlineSingleSample.BamToGvcf.skip_reblocking": true
+```
+
+The [Reblocking workflow](https://github.com/broadinstitute/warp/blob/master/pipelines/broad/dna_seq/germline/joint_genotyping/reblocking/ReblockGVCF.wdl) calls the GATK ReblockGVCF tool and uses the arguments:
+
+```WDL
+-do-qual-approx -floor-blocks -GQB 20 -GQB 30 -GQB 40 
+```
+The following summarizes how reblocking affects the WGS GVCF and downstream tools compared to the GVCF produced with HaplotypeCaller:
+
+
+1. Reblocked GVCFs are currently incompatible with the joint genotyping tool GenotypeGVCFs, but are compatible with GnarlyGenotyper which you can specify in the JSON configuration for the WARP JointGenotyping WDL workflow. 
```

Comment:
Laura's latest PR included the ability to run joint calling on reblocked gvcfs with both genotypegvcfs and gnarly

Reply:
Removed this line.